client/verta/tests/test_data_types.py

```diff
@@ -71,6 +71,18 @@ def test_discrete_histogram_numpy(self):
             },
         }
 
+    def test_missing_buckets(self):
+        one = data_types.DiscreteHistogram(
```

Comment:
Instead of `import numpy as np`
```suggestion
        np = pytest.importorskip("numpy")

        one = data_types.DiscreteHistogram(
```